src/docker-utils.coffee

```diff
@@ -101,27 +101,30 @@ do ->
 		Promise.using readLockImages(), ->
 			dockerProgress.pull(image, onProgress)
 
-	supervisorTag = config.supervisorImage
```

Comment:
I think it makes sense to keep this but as `supervisorTag = normalizeRepoTag(config.supervisorImage)` to avoid repeating the normalisation work each time (it might also be worth making the normalisation/splitting stuff return synchronously, as it doesn't actually need anything async)